src/NRediSearch/Query.cs

```diff
@@ -218,6 +225,17 @@ internal void SerializeRedisArgs(List<object> args)
                 args.Add((SortAscending ? "ASC" : "DESC").Literal());
             }
 
+            if (Scoring != null)
+            {
+                args.Add("SCORER".Literal());
+                args.Add(Scoring);
+
+                if (ExplainScore)
```

Comment:
@jjchiw `EXPLAINSCORE` should stand on its own and is not only limited to cases where `SCORER` is used

Reply:
I think `EXPLAINSCORE` literal it depends on `SCORER` (https://oss.redislabs.com/redisearch/Commands.html#ftsearch)

```
FT.SEARCH {index} {query} [NOCONTENT] [VERBATIM] [NOSTOPWORDS] [WITHSCORES] [WITHPAYLOADS] [WITHSORTKEYS]
  [FILTER {numeric_field} {min} {max}] ...
  [GEOFILTER {geo_field} {lon} {lat} {radius} m|km|mi|ft]
  [INKEYS {num} {key} ... ]
  [INFIELDS {num} {field} ... ]
  [RETURN {num} {field} ... ]
  [SUMMARIZE [FIELDS {num} {field} ... ] [FRAGS {num}] [LEN {fragsize}] [SEPARATOR {separator}]]
  [HIGHLIGHT [FIELDS {num} {field} ... ] [TAGS {open} {close}]]
  [SLOP {slop}] [INORDER]
  [LANGUAGE {language}]
  [EXPANDER {expander}]
  [SCORER {scorer}] [EXPLAINSCORE]
  [PAYLOAD {payload}]
  [SORTBY {field} [ASC|DESC]]
  [LIMIT offset num]
```